src/ActionBmwCodeCell.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { html } from "htm/preact";
+import { BlissSymbol, BciAvId } from "./BlissSymbol";
+import "./ActionBmwCodeCell.scss";
+
+function debugProps(x) {
```

Comment:
Please remove the debugging function before the merge. Thanks.

Reply:
Can you clarify?  Do you mean remove just the comment, "`// For debugging`"?  Or, the variable `fetchCount` and the `console.debug()` where the `fetchCount` value is logged?

I have checked the fetch count periodically and only once was it greater than one, which actually led to uncovering a bug.  But since then, it is always been "1".  I'm tempted to remove it at this point.